numalogic/connectors/druid/_druid.py

```diff
@@ -70,6 +70,10 @@ def build_params(
     intervals = [f"{start_dt.isoformat()}/{end_dt.isoformat()}"]
     dimension_specs = map(lambda d: DimensionSpec(dimension=d, output_name=d), dimensions)
 
+    config_ids = ""
+    if filter_pairs:
+        config_ids = list(filter_pairs)
```

Comment:
```suggestion
    config_ids = list(filter_pairs) if filter_pairs else ""
```

Reply:
Actually, filter pairs is always a dict, right? In that case, just do
```suggestion
    context = {
       ...,
       configIds: list(filter_pairs)
    }
    
```